ocw/lib/gce.py

```diff
@@ -142,6 +154,16 @@ def cleanup_all(self) -> None:
         self.cleanup_subnetworks()
         self.cleanup_networks()
 
+    def cleanup_blobs(self) -> None:
```

Comment:
I would add here check if bucket is defined and if not do early exit

Reply:
Fixed in the `cleanup_all` method.  Otherwise I would have to make the `__bucket` attribute non-private to properly mock it in the tests.